src/runtime/runtime.js

```diff
@@ -717,7 +717,17 @@ export class ConfiguredRuntime {
   getEntitlements(encryptedDocumentKey) {
     return this.entitlementsManager_
       .getEntitlements(encryptedDocumentKey)
-      .then(entitlements => entitlements.clone());
+      .then(entitlements => {
+        // Auto update internal things tracking the user's current SKU.
+        let sku = null;
+        if (entitlements) {
+          try {
+            sku = entitlements.getSingleEntitlement().getSku();
+          } catch (ex) {}
+        }
+        this.analyticsService_.setSku(sku);
```

Comment:
Is logging this info useful? If so, may as well log all the skus?

Reply:
Currently the logs do not know if you are already subscribed or not.  It is useful to know if you were already subscribed (give providing the SKUs you are subscribed to).

I can log all of them though.  That makes sense.